mojave/items/clothing/glasses/glasses.dm

```diff
@@ -0,0 +1,15 @@
+/obj/item/clothing/glasses/ms13
+	name = "generic ms13 glasses"
+	desc = "this doesn't even exist"
+	icon = 'mojave/icons/objects/clothing/glasses.dmi'
+	worn_icon = 'mojave/icons/mob/clothing/eyes.dmi'
+
+/obj/item/clothing/glasses/ms13/goggles
+	name = "thick goggles"
+	desc = "A nice thick pair of goggles with a questionably attached band around it. Good for keeping debris out of your eyes, wether it be dust or viscera."
+	icon_state = "biker"
+	inhand_icon_state = "sunglasses"
+
+/obj/item/clothing/glasses/ms13/goggles/biker
```

Comment:
This just uses the same exact icons as the base goggles?

Reply:
For now at least until there was something different. I'll remove it now that the prospectors got thanos snapped under my nose because you're a TOOL.